test/e2e/testdata/extract-event-time-from-payload.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
 spec:
   vertices:
     - name: in
+      containerTemplate:
```

Comment:
Please clean up.

Reply:
there is no harm in running e2e tests in debug mode

Reply:
Personally I prefer having a standard when developing e2e tests. I see some of the e2e test cases turn on debug mode and most don't so I am thinking we should make it consistent. No strong opinion though.